SPECS/mstflint/mstflint.spec

```diff
@@ -1,15 +1,14 @@
-Summary:        Mellanox firmware burning tool
+Summary:        Mellanox firmware burning application
 Name:           mstflint
-Version:        4.21.0
-Release:        4%{?dist}
+Version:        4.26.0
+Release:        1%{?dist}
 License:        GPLv2 OR BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Applications/System
 URL:            https://github.com/Mellanox/%{name}
-Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
-Patch4:         add-default-link-flags-for-shared-libraries.patch
-Patch6:         replace-mlxfwreset-with-mstfwreset-in-mstflint-message.patch
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-%{release}/%{name}-%{version}-%{release}.tar.gz
+ExclusiveArch: i386 i486 i586 i686 x86_64 ia64 ppc ppc64 ppc64le arm64 aarch64
```

Comment:
This could be reduced to the architectures we actually support, `x86_64` and `aarch64`